homeassistant/components/conversation/http.py

```diff
@@ -165,11 +160,11 @@ async def websocket_list_sentences(
     hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict
 ) -> None:
     """List custom registered sentences."""
-    agent = hass.data[DATA_DEFAULT_ENTITY]
+    manager = get_agent_manager(hass).default
 
     sentences = []
-    for trigger_data in agent.trigger_sentences:
-        sentences.extend(trigger_data.sentences)
+    for trigger_details in manager.triggers_details:
+        sentences.extend(trigger_details.sentences)
```

Comment:
These are all great changes that even if the split is not feasible, we should merge.